crates/compiler/test_gen/benches/list_map.rs

```diff
@@ -83,6 +88,9 @@ pub fn criterion_benchmark(c: &mut Criterion) {
         b.iter(|| unsafe {
             let mut main_result = RocCallResult::default();
 
+            // the roc code will dec this list, so inc it first so it is not free'd
+            std::mem::forget(input.clone());
```

Comment:
a consequence is that map cannot work in-place here (that is not implemented at the moment either)